ui/src/dashboards/reducers/cellEditorOverlay.ts

```diff
@@ -38,33 +39,37 @@ export const initialState = {
 export default (state = initialState, action: Action): CEOInitialState => {
   switch (action.type) {
     case ActionType.ShowCellEditorOverlay: {
-      const {
-        cell,
-        cell: {colors},
-      } = action.payload
-
-      const thresholdsListType = getThresholdsListType(colors)
-      const thresholdsListColors = validateThresholdsListColors(
-        colors,
-        thresholdsListType
-      )
-      const gaugeColors = validateGaugeColors(colors)
+      const {cell} = action.payload
 
       const tableOptions = getDeep<TableOptions>(
         cell,
         'tableOptions',
         initializeOptions(CellType.Table)
       )
 
-      const lineColors = validateLineColors(colors)
-
+      const colors = (cell as any).colors
```

Comment:
I think a simpler approach that will avoid coercing the cell type as `any` here is to the cell in the `ShowCellEditorOverlay` as `Partial<Cell>` which will make all keys optional or make `colors` an optional parameter.